dags/synapse-datasets-created-7-days.py

```diff
@@ -50,6 +50,7 @@
 }
 
 SYNAPSE_RESULTS_TABLE = "syn64951484"
+SYNAPSE_PUBLIC_RESULTS_TABLE = "syn64951485" #TODO
```

Comment:
Can we see the table?

Reply:
Yes, I just created the table and ran the DAG. You can see the result here: https://www.synapse.org/Synapse:syn69855119/tables/